2022-01-19 11:52:32,394 INFO    Thread-2  :25398 [wandb_setup.py:_flush():71] setting env: {'entity': 'amazonproject', 'project': 'Amazon-Reviews', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project', 'sweep_id': 'va6i92r5', 'run_id': '3liq254j', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/sweep-va6i92r5/config-3liq254j.yaml'}
2022-01-19 11:52:32,395 INFO    Thread-2  :25398 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-19 11:52:32,395 INFO    Thread-2  :25398 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220119_115232-3liq254j/logs/debug.log
2022-01-19 11:52:32,396 INFO    Thread-2  :25398 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220119_115232-3liq254j/logs/debug-internal.log
2022-01-19 11:52:32,396 INFO    Thread-2  :25398 [wandb_init.py:init():404] calling init triggers
2022-01-19 11:52:32,396 INFO    Thread-2  :25398 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 200, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.01, 'num_workers': 4, 'optimizer': 'adam'}
config: {}
2022-01-19 11:52:32,396 INFO    Thread-2  :25398 [wandb_init.py:init():460] starting backend
2022-01-19 11:52:32,397 INFO    Thread-2  :25398 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-19 11:52:32,433 INFO    Thread-2  :25398 [backend.py:ensure_launched():216] starting backend process...
2022-01-19 11:52:32,461 INFO    Thread-2  :25398 [backend.py:ensure_launched():221] started backend process with pid: 25406
2022-01-19 11:52:32,476 INFO    Thread-2  :25398 [wandb_init.py:init():469] backend started and connected
2022-01-19 11:52:32,484 INFO    Thread-2  :25398 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 200, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.01, 'num_workers': 4, 'optimizer': 'adam'}
2022-01-19 11:52:32,500 INFO    Thread-2  :25398 [wandb_init.py:init():533] updated telemetry
2022-01-19 11:52:32,812 INFO    Thread-2  :25398 [wandb_init.py:init():563] communicating current version
2022-01-19 11:52:37,821 INFO    Thread-2  :25398 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-19 11:52:46,823 INFO    Thread-2  :25398 [wandb_init.py:init():606] starting run threads in backend
2022-01-19 11:52:51,846 INFO    Thread-2  :25398 [wandb_run.py:_console_start():1810] atexit reg
2022-01-19 11:52:51,848 INFO    Thread-2  :25398 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-19 11:52:51,849 INFO    Thread-2  :25398 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-19 11:52:51,851 ERROR   Thread-2  :25398 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-19 11:52:51,860 INFO    Thread-2  :25398 [wandb_init.py:init():633] run started, returning control to user process
